src/averageAges.js

```diff
@@ -55,7 +66,18 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const children = onlyWithSon
+    ? people.filter(child => people
+      .find(person => person.name === child.mother))
+      .filter(person => person.sex === 'm')
```

Comment:
Keep correct code style. `find` and `filter` are to different entities, so they should not be on the same level. 

```suggestion
    ? people
      .filter(child => people
        .find(person => person.name === child.mother))
      .filter(person => person.sex === 'm')
```